code/game/machinery/computer/cloning.dm

```diff
@@ -67,8 +66,8 @@
 			else if(!. && pod.is_operational && !(pod.occupant || pod.mess) && pod.efficiency > 5)
 				. = pod
 
-/proc/grow_clone_from_record(obj/machinery/clonepod/pod, datum/data/record/R, experimental)
-	return pod.growclone(R.fields["name"], R.fields["UI"], R.fields["SE"], R.fields["mindref"], R.fields["last_death"], R.fields["mrace"], R.fields["features"], R.fields["factions"], R.fields["bank_account"], R.fields["traumas"], R.fields["body_only"], experimental)
+/proc/grow_clone_from_record(obj/machinery/clonepod/pod, datum/record/cloning/R, experimental)
+	return pod.growclone(R.name, R.uni_identity, R.SE, R.mind, R.last_death, R.species, R.dna.features, R.factions, R.mind.account_id, R.traumas, R.body_only, experimental)
```

Comment:
```suggestion
	return pod.growclone(R.name, R.uni_identity, R.SE, R.mind, R.last_death, R.species, R.dna.features, R.factions, R.mind.account_id, R.traumas, R.body_only, experimental)
```

`R.dna.features` can be `R.resolve_dna_features()` that does `var/datum/dna/dna = weakref_dna.resolve(); return dna.features` in this case
mind is similar too.

You should change those direct access codes into weakref resolve.